app/views/home/about.html.erb

```diff
@@ -15,5 +15,5 @@ commercial exploitation and legal challenge.") %></li>
 that seeks to promote scholarship on fanworks and practices.") %></li>
     </ul>
   </p>
-  <p><%= ts("You can find out more about the OTW and its projects at its website,") %> <%= link_to ts("transformativeworks.org"), "http://transformativeworks.org" %><%= ts(", and learn about how your financial support is vital to the continuation and expansion of the OTW's work on its") %> <%= link_to ts("FAQ page"), "http://transformativeworks.org/faq-page" %>.</p>
+  <p><%= ts("You can find out more about the OTW and its projects at its website,") %> <%= link_to ts("transformativeworks.org"), "http://transformativeworks.org" %><%= ts(", and learn about how your financial support is vital to the continuation and expansion of the OTW's work on its") %> <%= link_to ts("FAQ page"), "http://transformativeworks.org/faq" %><%= ts(". If you have a media or research question, please contact the") %> <%= link_to ts("Communications team"), "http://transformativeworks.org/contact_us/concomm" %>.</p>
```

Comment:
I would like us to use arguments to the ts call to allow the translators the most flexibility we can give them. For example I think the first one should look something like:

ts("You can find out more about the OTW and its projects at its website, %{transformative_works} , and learn about how your financial support is vital to the continuation and expansion of the OTW's work on its %{faq}.", :transformative_works => link_to(ts("transformativeworks.org"), "http://transformativeworks.org") , :faq => link_to ( ts("FAQ page"), "http://transformativeworks.org/faq" ) )

Note this is completely untested and just so hopefully you can understand what I mean.